rosbag2_test_common/include/rosbag2_test_common/service_manager.hpp

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 Sony Group Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROSBAG2_TEST_COMMON__SERVICE_MANAGER_HPP_
+#define ROSBAG2_TEST_COMMON__SERVICE_MANAGER_HPP_
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "rclcpp/rclcpp.hpp"
+
+namespace rosbag2_test_common
+{
+class ServiceManager
+{
+public:
+  ServiceManager()
+  : pub_node_(std::make_shared<rclcpp::Node>(
+        "service_manager_" + std::to_string(rclcpp::Clock().now().nanoseconds()),
+        rclcpp::NodeOptions().start_parameter_event_publisher(false).enable_rosout(false)))
```

Comment:
Comments from MichaelOrlov

Nitpick. Shall we add .start_parameter_services(false) for node parameters as well?